src/core/server/metrics/event_loop_delays/event_loop_delays_monitor.test.ts

```diff
@@ -39,10 +39,20 @@ describe('EventLoopDelaysMonitor', () => {
     expect(eventLoopDelaysMonitor['loopMonitor'].percentile).toHaveBeenNthCalledWith(3, 95);
     expect(eventLoopDelaysMonitor['loopMonitor'].percentile).toHaveBeenNthCalledWith(4, 99);
 
-    // mocked perf_hook returns `mocked.createHistogram()`.
+    // mocked perf_hook returns `mocked.createNsHistogram()` that returns data in ns.
+    // The `collect` function returns the data in ms.
     // This ensures that the wiring of the `collect` function is correct.
     const mockedHistogram = mocked.createHistogram();
-    expect(histogramData).toEqual(mockedHistogram);
+
+    expect(histogramData.min).toEqual(nsToMs(mockedHistogram.min));
```

Comment:
I couldn't come up with an elegant way to test all of this that was still explicit enough for us to glance over and realize there's a unit conversion taking place.

Since the units already threw us once, I'm erring on the side of caution here.